mobile/scripts/buildStatusGo.sh

```diff
@@ -30,12 +30,18 @@ make generate V=3 SHELL=/bin/sh
 mkdir -p build/bin/statusgo-lib
 go run cmd/library/*.go > build/bin/statusgo-lib/main.go
 
-GOFLAGS="" CGO_ENABLED=1 GOOS="$OS" GOARCH="$GOARCH" \
-	go build \
-		-buildmode="$BUILD_MODE" \
-		-tags 'gowaku_no_rln nowatchdog disable_torrent' \
-		-ldflags="-X github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/metrics.EnabledStr=true" \
-		-o "$LIB_DIR/libstatus.$LIB_EXT" \
-		./build/bin/statusgo-lib
+# status-go requires clang from Xcode
+(
+	export CC="/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang"
+	export CGO_CFLAGS="-arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS18.0.sdk -miphoneos-version-min=16.0 -fembed-bitcode"
```

Comment:
Why are we hardcoding this for Xcode? Wat about other platforms?

Reply:
ah good question, we need this only in case of IOS builds, perhaps there is a better spot to keep this in